services/provider/server/server.go

```diff
@@ -252,14 +252,22 @@ func (s *OCSProviderServer) GetStorageConfig(ctx context.Context, req *pb.Storag
 			isConsumerMirrorEnabled,
 		)
 
+		systemAtteributes := &pb.SystemAttributes{
+			SystemInMaintenanceMode: inMaintenanceMode,
+			MirrorEnabled:           isConsumerMirrorEnabled,
+			DesiredRbdDriver:        &pb.RbdDriverRequirements{},
+			DesiredCephFsDriver:     &pb.CephFsDriverRequirements{},
+		}
+
+		if storageCluster.Spec.NFS != nil && storageCluster.Spec.NFS.Enable {
+			systemAtteributes.DesiredNfsDriver = &pb.NfsDriverRequirements{}
```

Comment:
what is the de-serialized form for w/ & w/o the message? Is it going to be compared to `nil` at client side? possible to check it as I forgot how it'll look like.

Reply:
Without the message it will be de-serialized as `nil` on the client side